### ISSUE TITLE
NPC aggro in sap

### ISSUE THREAD

**Issue by rechtw** [elysium-project/tbc-bug-tracker#667]
Sapped NPC's aggro when i'm in stealth and using Pick Pocket, i think it's ridiculous, i can't even eat in stealth because of that

**Comment by McMittens** [comment]
I could not reproduce this. Missing sap doesn't trigger combat and failing to pickpocket should start a fight. I was able to sap and pickpocket successfully without triggerng combat.

What exactly happened for you?

**Comment by rechtw** [comment]
Sorry my bad english, If you missing pickpocket on sapped mob, he goes
aggro to you in invisible, it's ridiculous, bacuase he's sapped, he don't
should to aggro on you, because he's sapped, and why .st command is work?
For what?

сб, 25 мая 2019 г., 17:39 McMittens <notifications@github.com>:

**Comment by McMittens** [comment]
https://wow.gamepedia.com/index.php?title=Pick_Pocket&oldid=1117932
old revision from early 2008

![huh](https://user-images.githubusercontent.com/46020700/58380003-97893e80-7fab-11e9-843f-6aaf3e6b892f.PNG)
![huh2](https://user-images.githubusercontent.com/46020700/58380004-99530200-7fab-11e9-9602-87078c6109f1.PNG)

So it isn't working correctly as in failing the pickpocket doesn't cancel your stealth as it should. The mob will aggro you even if sapped, but since he can't follow you bc of the sap stun you can outrage them to end the fight.

So in conclusion:
Currently if you fail a pickpocket you enter combat with the NPC, this is the intended behaviour, but it doesnt automatically break stealth.

Stealth should always break on a failed pickpocket.

**Comment by McMittens** [comment]
failed pickpocket now breaks stealth
